cloud/amazon/route53.py

```diff
@@ -511,7 +526,10 @@ def main():
     except boto.route53.exception.DNSServerError, e:
         txt = e.body.split("<Message>")[1]
         txt = txt.split("</Message>")[0]
-        module.fail_json(msg = txt)
+        if "but it already exists" in txt:
```

Comment:
The error message is from amazon route 53 (and it's mentioned in theirs documentation http://docs.aws.amazon.com/Route53/latest/APIReference/API_ChangeResourceRecordSets_Errors.html) 

Before this change all requests trying to add existing entry were finished with fail_json. 

I don't like this solution too, if you have any idea how it can be done in clean way basing on the amazon docs, I can change it.